packages/ui/src/theme/css/component/button.scss

```diff
@@ -227,6 +227,11 @@
   }
 }
 
+.amplify-button .amplify-button__loader-wrapper {
```

Comment:
Should this selector just be `.amplify-button__loader-wrapper` to keep it at 1 classname?

Reply:
@dbanksdesign Yea, I do try that at first.

<img width="648" alt="Screen Shot 2022-02-17 at 9 57 18 AM" src="https://user-images.githubusercontent.com/40295569/154542189-5f4977fe-f030-42b9-bf7e-268e24293392.png">

But the order matters here. The `Flex` styling cannot be overridden if we just use 1 classname